imagej/imagej-ops2/src/main/java/net/imagej/ops2/filter/convolve/ConvolveFFTF.java

```diff
@@ -181,10 +183,10 @@ public Computers.Arity2<RandomAccessibleInterval<I>, RandomAccessibleInterval<K>
 	 * @return the output
 	 */
 	@Override
-		public RandomAccessibleInterval<FloatType> apply(RandomAccessibleInterval<I> t, RandomAccessibleInterval<K> u, ExecutorService es) {
+		public RandomAccessibleInterval<FloatType> apply(RandomAccessibleInterval<I> input, RandomAccessibleInterval<K> kernel, ExecutorService executorService) {
 			OutOfBoundsFactory<I, RandomAccessibleInterval<I>> obfInput = new OutOfBoundsConstantValueFactory<>(Util
-					.getTypeFromInterval(t).createVariable());
-			return convolveOp.apply(t, u, null, obfInput, new FloatType(), new ComplexFloatType(), es);
+					.getTypeFromInterval(input).createVariable());
+			return convolveOp.apply(input, kernel, new FloatType(), new ComplexFloatType(), executorService, null, obfInput);
 		}
```

Comment:
Do you think it's worth keeping this Op? We could just make more parameters Optional above.

Reply:
I'm ok leaving it since it ties the complex and out types together.